src/sql/src/func.rs

```diff
@@ -2342,6 +2342,14 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
         "pg_postmaster_start_time" => Scalar {
             params!() => UnmaterializableFunc::PgPostmasterStartTime => TimestampTz, 2560;
         },
+        "pg_relation_size" => Scalar {
+            params!(RegClass, String) => sql_impl_func("0::pg_catalog.int8") => Int64, 2332;
+            params!(RegClass) => sql_impl_func("0::pg_catalog.int8") => Int64, 2325;
+            params!(Oid) => sql_impl_func("0::pg_catalog.int8") => Int64, 2325;
```

Comment:
The postgres docs only specify `pg_relation_size(regclass, string)` and `pg_relation(regclass)`, however I found dbeaver was throwing an error trying to call `pg_relation(oid)`. `regclass` can always be trivially cast to `oid` so this seems safe, but I don't know if there's a way to better express that here.

Reply:
Well, looks like this causers a test to fail.